Assignment3/Populate.txt

```diff
@@ -0,0 +1,30 @@
+public class Populate
+{
+    public void updateOfOpportunities(List<Opportunity> opprList)
```

Comment:
This method should be called when an opportunity is updated. Have you tested this scenario?

Reply:
Yes. It works.